styles.css

```diff
@@ -0,0 +1,465 @@
+  transform: rotate(-45deg) translate(-4px, 4px);
+}
+
+/* <a href="https://www.freepik.com/free-photo/dark-concrete-texture-background_9053906.htm#query=texture%20background&position=10&from_view=keyword&track=ais">Image by kbza</a> on Freepik */
+.presentation {
+  padding: 3rem 1rem;
+}
+
+h1 {
+  color: #f24822;
+  font-weight: 700;
+  font-size: 3rem;
+  padding: 1rem 0%;
+}
+
+.tagline {
+  color: #f24822;
+  font-size: 2rem;
+  font-weight: 600;
+  margin: 0%;
+}
+
+.description {
+  font-size: 1.3rem;
+  font-weight: 400;
+}
+
+.details-presentation { padding: 2rem 0%; }
+
+.schedule {
+  font-size: 2rem;
+  font-weight: 600;
+}
+
+.location {
+  color: #838080;
+  font-size: 1.4rem;
+  font-weight: 600;
+}
+
+.section-title {
+  font-size: 1.6rem;
+  text-align: center;
+}
+
+h2 {
+  padding: 1rem;
+}
+
+.events {
+  background-image: url(images/events-bkg.jpeg);
+  padding: 3rem 0%;
+}
+
+.events .container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+}
+
+#events-section-title { color: hsl(0, 0%, 100%); }
+
+.event-title {
+  color: #f24822;
+  margin: 0%;
+}
+
+.events-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.event-card {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  background-color: #beb7b717;
+  gap: 0.4rem;
+  padding: 0.5rem;
+  transition: all 0.6s;
+}
+
+.event-card:hover {
+  transform: scale(1.1);
+}
+
+.event-card img {
+  width: 40%;
+}
+
+.event-description {
+  color: #fff;
+  margin: 0%;
+}
+
+.button {
+  background-color: #f1370d;
+  font-weight: 500;
+  transition: 0.3s ease;
+  color: #fff;
+  border: none;
+  padding: 5%;
+  border-radius: 3px;
+}
+
+.button:hover {
+  transform: scale(1.1);
+  box-shadow: 1px 2px 3px #50505044;
+  background-color: #c92703;
+}
+
+.dishes-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.products .container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.products .container h2 { padding: 0.4rem; }
+
+.card-product {
+  display: flex;
+  align-items: start;
+  justify-content: space-around;
+  padding: 1rem;
+  gap: 0.8rem;
+  background-image: url(images/product-background.jpg);
+  background-repeat: no-repeat;
+  background-size: 20% 40%;
+  background-position: 0% 0%;
+  transition-duration: 0.6s;
+}
+
+.card-product:hover {
+  transform: translateX(12px);
+}
+
+.card-product img {
+  width: 50%;
+  height: 75%;
+}
+
+.product-description {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  justify-content: space-around;
+}
+
+.product-description h3 {
+  margin: 0%;
+}
+
+#moreBtn {
+  color: black;
+  background-color: #fff;
+  border: #3e3e3e 1px solid;
+}
+
+.desktop {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .container {
+    max-width: 750px;
+    margin: auto;
+    padding: 0%;
+  }
+
+  .hamburger {
+    display: none;
+  }
```

Comment:
- [ ]  Kindly avoid code repetition in accordance to DRY principle, you've done the same in line 464. 👍